#diffx: encoding=utf-8, version=1.0
#.change:
#..file:
#...meta: format=json, length=269
{
    "path": "/readme",
    "revision": {
        "old": "123"
    },
    "svn": {
        "properties": {
            "myproperty": {
                "new": "new value",
                "old": "old value"
            }
        }
    },
    "type": "svn:properties"
}
#...diff: length=266
Index: readme
===================================================================
--- (revision 123)
+++ (working copy)
Property changes on: .
-------------------------------------------------------------------
Modified: myproperty
## -1 +1 ##
-old value
+new value
